intro-to-tests/fizz_buzz.instructor.md

```diff
@@ -11,8 +11,7 @@ The live code repo: [TDD Ping Pong /Fizz Buzz](https://github.com/AdaGold/tdd-pi
 The suggested way to run this live code is one instructor writes a test and then the other instructor implements code to make the test pass. The second instructor then writes the next test.
 
 A `requirements.txt` is not currently provided. Here are a few suggestions for how to run this exercise related to the `requirements.txt`.
-- We could not use venv and use the globally installed pytest for this exercise
 - We could use venv and install pytest and create a requirements.txt
 - We could create a `requirements.txt` in the repo and model creating a venv and installing the requirements
 
-In C15 and C16 we introduced `requirements.txt` and `venv` with the first project.
\ No newline at end of file
+In C15 and C16 we introduced `requirements.txt` and `venv` with the first project. Starting with C21, introduced `requirements.txt` and `venv` in precourse due to Python no longer allowing the global installation of packages. 
```

Comment:
I'm wondering if this historical context is useful in the resources for the activity? If others are okay with it, I'm good removing this chunk-we have the git commit history if we need to look at when we made changes to the lesson.

```suggestion
```

Reply:
Yeah, I'm fine with that. Plus, we have it in Asana as well in the curriculum tasks. I can remove it.